prospector/client/cli/console_report.py

```diff
@@ -15,22 +18,18 @@ def format_annotations(commit: CommitWithFeatures) -> str:
 
         return out
 
-    print("-" * 80)
```

Comment:
This is the only place where the output is not about "optional" logging, it is the actual functionality of the tool, I cannot imagine a use case in which this output would be silenced; I would suggest reverting the changes (for this function only)

Reply:
I merged as is, will revert the prints for this fucntion as a separate commit later